app/views/places/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2><%= current_user.name %>!</h2>
+
+<p>
+<strong>Email:</strong> <%= current_user.email %>
+<%= link_to 'Places', user_places_path(current_user.id) %>
+<%= link_to 'Create Place', new_user_place_path(current_user.id) %>
+</p>
+
+
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Places</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Place id</th>
+      <th>Comment</th>
+      <th>Owner_id</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+  <% @places.each do |place| %>
+      <tr>
+        <td><%= place.id %></td>
+        <td><%= place.comments %></td>
+        <td><%= place.owner_id %></td>
+
+        <td><%= link_to '<i class="fa fa-caret-left fa-2x"></i>'.html_safe, profile_path%><td>
```

Comment:
You can pass a block to the `link_to` helper so you don´t need to do the `html_safe` stuff. Ie:
```ruby
link_to profile_path do
  <i class="fa fa-caret-left fa-2x"></i>
end
```